src/ess/amor/resolution.py

```diff
@@ -29,7 +29,7 @@ def wavelength_resolution(
     :
         The wavelength resolution variable, as standard deviation.
     """
-    return fwhm_to_std(chopper_separation / (L1 + L2))
+    return fwhm_to_std(sc.abs(chopper_separation) / (L1 + L2))
```

Comment:
Is this a bugfix? If yes, please add a test!

Reply:
Not a bugfix, just defensive. Do you want a test anyway?

Reply:
Not as part of this PR in that case. But the individual functions should be unit-tested anyway.